clang/test/SemaCXX/PR68542.cpp

```diff
@@ -0,0 +1,20 @@
+// RUN: %clang_cc1 -verify -std=c++20 -fsyntax-only %s
+
+// expected-note@+2{{candidate constructor (the implicit move constructor) not viable: no known conversion from 'int' to 'S &&' for 1st argument}}
```

Comment:
Can you use the bookmark feature to put these notes next to the error they are associated with?  I'm not positive where the errors are here.